pyproject.toml

```diff
@@ -25,15 +25,51 @@ dependencies = [
     "flatten_dict",
     "h5py",
     "numpy",
-    "mujoco==3.1.2",
-    "dm-control==1.0.16",
+    "dm-control==1.0.28",
     "Pillow",
     "pink-noise-rl",
     "packaging",
     "gitpython"
 ]
 [project.optional-dependencies]
 examples = ["mink[examples]; python_version > '3.9'"]
+standard = [
```

Comment:
it would be better to have this as default to simplify installation

Reply:
done